.eslintrc.cjs

```diff
@@ -463,9 +464,11 @@ const rules = {
   'jest/prefer-each': 1,
   'jest/prefer-equality-matcher': 1,
   'jest/prefer-expect-assertions': 0,
-  'jest/prefer-expect-resolves': 0,
+  'jest/prefer-expect-resolves': 1,
```

Comment:
I think we can enable it. Wdyt?
https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-expect-resolves.md